src/products/categories/categories.controller.ts

```diff
@@ -26,8 +26,32 @@ export class CategoriesController {
     
     @Post()
     Create(@Body()item:Category):boolean{
-        //this.categories.push(item);
+        this.CategorysRepository.save(item);
         console.log(item);
         return true;        
     }
+
+    @Patch(":id")
+    async Update(@Param() params, @Body()item:Category) {
+        // let oldItem = await this.CategorysRepository.findOneBy({id:params.id});
+        // oldItem = item;
+        // if(oldItem){
+        //     await this.CategorysRepository.save({id: oldItem.id, ...item});
+        // }
```

Comment:
commented code: Please remove that code if it is not planned to be used in the future.
Otherwise, please add todo to define why this code is left and commented.